src/EnergyPlus/PoweredInductionUnits.cc

```diff
@@ -1643,41 +1652,32 @@ namespace PoweredInductionUnits {
 		// Set the mass flow rates
 		if ( UnitOn ) {
 			// unit is on
-			// Special controls if FanOnFlowFrac = 0.0
+			// Calculate if reheat is needed
 			bool ReheatRequired = false;
-			if (PIU(PIUNum).FanOnFlowFrac <= 0.0) {
-				// Calculate if reheat is needed
-				Real64 qMinPrimary = PriAirMassFlowMin * ( CpAirZn * min( -SmallTempDiff, ( Node( PriNode ).Temp - Node( ZoneNode ).Temp ) ) );
-				if (  qMinPrimary < QToHeatSetPt ) ReheatRequired = true;
-			}
+			Real64 qMinPrimary = PriAirMassFlowMin * ( CpAirZn * min( -SmallTempDiff, ( Node( PriNode ).Temp - Node( ZoneNode ).Temp ) ) );
+			if (  qMinPrimary < QToHeatSetPt ) ReheatRequired = true;
 
 			if ( ! PriOn ) {
 				// no primary air flow
 				PriAirMassFlow = 0.0;
-				// PIU fan off if FanOnFlowFrac is 0.0 and there is no heating load, also reset fan flag if fan should be off
-				if ( PIU( PIUNum ).FanOnFlowFrac <= 0.0 ) {
-					if ( QZnReq <= SmallLoad ) {
-						SecAirMassFlow = 0.0;
-						PIUTurnFansOn = false;
-					} else {
-						SecAirMassFlow = PIU( PIUNum ).MaxSecAirMassFlow;
-						PIUTurnFansOn = ( DataHVACGlobals::TurnFansOn || DataHVACGlobals::TurnZoneFansOnlyOn );
-					}
+				// PIU fan off if there is no heating load, also reset fan flag if fan should be off
+				if ( QZnReq <= SmallLoad ) {
+					SecAirMassFlow = 0.0;
+					PIUTurnFansOn = false;
 				} else {
 					SecAirMassFlow = PIU( PIUNum ).MaxSecAirMassFlow;
+					PIUTurnFansOn = ( DataHVACGlobals::TurnFansOn || DataHVACGlobals::TurnZoneFansOnlyOn );
```

Comment:
This looks like the load overrides the avail managers, that sounds backwards. If there is no load, turn off. If there is a load, and avail managers say to turn on, then turn on. I guess my question here would be why would the avail manager tell the fan to turn on if there were no load? Because I just wanted to...maybe?

Reply:
Because the avail manager signal is global to all zone fans on the same airloop regardless of which zones may need heat.  So, it's like an "and" if the avail manger doesn't send a CycleOnZoneFansOnly, then the fan won't come on, regardless of the zone load.  But if the avail manager says go *and* the zone needs heat, then, yes, fire it up.  If you prefer that the availability manager be king - then we could change CycleOnZoneFansOnly to an array by zones and let the availability manager set this zone-by-zone.

Reply:
OK, so the PIUs don't have avail managers like the other zone equipment. So yes, it's a global flag.

    ZoneHVAC:FourPipeFanCoil,
       A15, \field Availability Manager List Name
          \note Enter the name of an AvailabilityManagerAssignmentList object.

But the revised documentation says:

    The secondary fan will run only when the Availability Schedule specified in the fan object is >0 or when it is overridden by an availability manager

I'm not saying it should or shouldn't act one way or another, and I think this is what the unit would do in reality, but what would a user say if they set an avail manager and the fan didn't turn on or off as they programmed? Maybe just a revision to the avail manager docs to say if no load for PIU then secondary fans won't turn on?